# HG changeset patch
# User Felix Janda <felix.janda@posteo.de>
# Date 1423171584 -3600
#      Thu Feb 05 22:26:24 2015 +0100
# Node ID 48a43c9da31a3ae083edd563be50fb04d9a5b99a
# Parent  2d344ef8da33717224d5a08570dc834ca2254f9f
# https://bugzilla.mozilla.org/show_bug.cgi?id=1151202
libstagefright: Fix compilation for systems without <sys/cdefs.h>

diff -r 2d344ef8da33 -r 48a43c9da31a media/libstagefright/system/core/include/cutils/properties.h
--- a/media/libstagefright/system/core/include/cutils/properties.h	Sun Feb 01 19:32:36 2015 +0100
+++ b/media/libstagefright/system/core/include/cutils/properties.h	Thu Feb 05 22:26:24 2015 +0100
@@ -17,7 +17,7 @@
 #ifndef __CUTILS_PROPERTIES_H
 #define __CUTILS_PROPERTIES_H
 
-#include <sys/cdefs.h>
+#include <sys/types.h>
 #include <stddef.h>
 
 #ifdef __cplusplus
